ariadne_codegen/client_generators/constants.py

```diff
@@ -52,3 +52,6 @@
 
 SCALARS_PARSE_DICT_NAME = "SCALARS_PARSE_FUNCTIONS"
 SCALARS_SERIALIZE_DICT_NAME = "SCALARS_SERIALIZE_FUNCTIONS"
+
+UNSET = "UNSET"
```

Comment:
This should be `class Unset:` because current implementation prevents use of  `"UNSET"` value in inputs.

Reply:
This constant is used by generator to keep track of variable's name with constant, I changed it to `UNSET_NAME`. `UNSET` used by the generated client is declared [here](https://github.com/mirumee/ariadne-codegen/blob/unset_values/ariadne_codegen/client_generators/dependencies/base_model.py#L15)
```py
class UnsetType:
    def __bool__(self) -> bool:
        return False


UNSET = UnsetType()
```

Reply:
Oh okay, that confused me a bit here.